examples/alternate-receive/alternate_receive.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    worker w1 {
+        fetch("https://postman-echo.com/get?worker=w1") -> w3;
```

Comment:
Shall we extract the result of the fetch call out to a variable for readability? Because the send syntax itself isn't something that's commonly used and this is an example.